src/cluster.h

```diff
@@ -50,7 +50,7 @@ namespace zim
     public:
       Cluster(std::shared_ptr<const Reader> reader, CompressionType comp, bool isExtended);
       CompressionType getCompression() const   { return compression; }
-      bool isCompressed() const                { return compression == zimcompZip || compression == zimcompBzip2 || compression == zimcompLzma; }
+      bool isCompressed() const                { return compression == zimcompZip || compression == zimcompBzip2 || compression == zimcompLzma || compression == zimcompZstd; }
```

Comment:
This beginning to be a bit long. Maybe test that compression is different that default and none would be shorter.

Reply:
Done